plop/component/index.js

```diff
@@ -27,9 +27,25 @@ module.exports = plop => {
       },
       {
         type: "add",
-        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.stories.mdx",
+        path: "src/components/{{properCase componentName}}/__stories__/{{properCase componentName}}.mdx",
```

Comment:
Why do some files have the `.stories.` and some don't?

Reply:
`.mdx` files shouldn't have `.stories.` prefix now, that's the storybook 7 law kinda - they just won't render
Rest of the stories related files is with `.stories.` prefix